RealmSwift-swift2.0/Tests/PerformanceTests.swift

```diff
@@ -361,7 +361,7 @@ class SwiftPerformanceTests: TestCase {
                 }
             }
         }
-        _ = realm.path
+        _ = realm
```

Comment:
This needs to actually access something on the object which the compiler can't prove doesn't have side effects.